app/src/main/res/layout/fragment_search_paginated.xml

```diff
@@ -3,7 +3,8 @@
   android:layout_width="match_parent"
   android:layout_height="match_parent"
   android:orientation="vertical"
-  android:paddingTop="@dimen/tiny_gap">
+  android:paddingTop="@dimen/tiny_gap"
+  android:background="?attr/mainBackground">
```

Comment:
Thanks for your cooperation and fast returns @Pratham2305 , the other problem about removing the fragment is solved after your last commit. It still works as it should. My other question is why you needed to add backgrounds to some .xml elements?

Reply:
Because the background color is not changed, after changing the theme from the settings. Try changing the white to the dark theme and You will notice the background and toolbar are still white but the title text color on the toolbar is changed, So I have added the background property similar to the titleTextcolor property.